src/no/runsafe/UserControl/database/PlayerDatabase.java

```diff
@@ -64,6 +64,31 @@ public ISchemaUpdate getSchemaUpdateQueries()
 
 		update.addQueries("ALTER TABLE player_db ADD COLUMN temp_ban datetime NULL");
 		update.addQueries("ALTER TABLE player_db ADD COLUMN uuid VARCHAR(255) NULL");
+
+		update.addQueries("ALTER TABLE player_db RENAME TO player_db_old");
+
+		update.addQueries( // Create new table based on player uuids instead of usernames.
+			"CREATE TABLE `" + getTableName() + "` (" +
+				"`uuid` varchar(255) NOT NULL," +

```

Comment:
I'm pretty sure there is a dedicated data type for uuid we can use rather than `varchar(255)` here.

Reply:
also, should we not still save their name and then update it when we see a change, so we won't need to do external lookups by uuid to find their name?